src/backend/mod.rs

```diff
@@ -146,7 +146,11 @@ pub(super) unsafe fn execute_gdb(
     use sallyport::item;
 
     match gdbcall {
-        item::Gdbcall { num, ret, .. } if *num == item::gdbcall::Number::OnSessionStart => {
+        item::Gdbcall {
+            num: item::gdbcall::Number::OnSessionStart,
```

Comment:
Might as well bring Gdbcall and Number into scope to avoid a lot of verbose namespacing.